ci/ci/github.py

```diff
@@ -140,8 +140,8 @@ def clone_or_fetch_script(repo):
 { RETRY_FUNCTION_SCRIPT }
 
 function clone() {{
-    rm -rf ./{*,.*}
-    git clone {shq(repo)} ./
+    rm -rf ./{{*,.*}}
```

Comment:
For my education, can you explain what this is doing? Is it matching hidden files?

Reply:
Yeah. `{..}` is a bash-ism for kind of map-expanding some values:
```
# ls a{bb,cc}d
ls: abbd: No such file or directory
ls: accd: No such file or directory
```

And then `.*` means all the hidden files